service/src/main/java/greencity/service/ubs/SuperAdminServiceImpl.java

```diff
@@ -405,9 +395,9 @@ public GetTariffServiceDto includeBag(Integer id) {
             throw new BadRequestException(ErrorMessage.BAG_WITH_THIS_STATUS_ALREADY_SET);
         }
         bag.setMinAmountOfBags(MinAmountOfBag.INCLUDE);
+        bag.setLocation(Location.builder().id(1L).build());
```

Comment:
Why do we set `id = 1L` for all requests?

Reply:
Maybe I was confused about SuperAdminServiceImpl with Test, sorry. I deleted it

Reply:
I deleted it, because in my Mock I forgot set Location id, it's working now